tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -1,4 +1,5 @@
 namespace.gluster:
+  gluster_brick_path: "/glusterfs"
```

Comment:
Your point makes sense.
I just had a discussion with shubhendu and RHSC folks(old gluster console)  and got inputs that it would be better to make it configurable. So am planning to move this ti gluster_integration conf file and have "tendrl_gluster" as default
@r0h4n what do you feel ?